gazebo/rendering/LensFlare.cc

```diff
@@ -392,6 +77,280 @@ namespace gazebo
 using namespace gazebo;
 using namespace rendering;
 
+//////////////////////////////////////////////////
+LensFlareCompositorListener::LensFlareCompositorListener(CameraPtr _camera, LightPtr _light)
```

Comment:
I think it would simplify the diff if we moved the implementation of the `LensFlareCompositorListener` methods back so they are above `LensFlarePrivate` again. The whitespace is negotiable, since we can ignore it with `?w=1` added to the URL

Reply:
thanks, the `?w=1` diff is much cleaner now!